tf_encrypted/keras/layers/normalization_test.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+import tf_encrypted as tfe
+from tf_encrypted.keras.testing_utils import agreement_test, layer_test
+
+np.random.seed(42)
+
+
+class TestBatchNormalization(unittest.TestCase):
+
+  def setUp(self):
+    tf.reset_default_graph()
+
+  def test_batchnorm_2d(self):
+    self._core_batchnorm([1, 4], axis=1)
+
+  def test_batchnorm_4d(self):
+    self._core_batchnorm([1, 1, 1, 3])
```

Comment:
I would suggest to have a test where `moving_mean_initializer` and `moving_variance_initializer` are not set to the defaults zeros and ones respectively.